index.html

```diff
@@ -256,6 +260,7 @@ <h3 id="out-of-scope">Out of Scope</h3>
               <li>Browser APIs</li>
               <li>Specific Implementations of DID Methods and Protocols</li>
               <li>"Solving Identity" on the Web</li>
+              <li>Defining a concrete DID method implementation</li>
```

Comment:
It is not clear to me how this additional line differs from the existing out of scope statement:
> Specific Implementations of DID Methods and Protocols